### DISCUSSION TITLE
URL doesn't point to the correct subheading on Chrome Browser

### DISCUSSION THREAD

**Discussion by kesava-karri** [TheOdinProject/theodinproject#4820] [General]
* In [this](https://www.theodinproject.com/lessons/node-path-javascript-form-validation-with-javascript#introduction) lesson's assignment where it redirects to [mdn's tutorial](https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation#validating_forms_using_javascript) on form validation doesn't show the subheading `Validating forms using JavaScript` instead opens the webpage from the start.

* **Note**: This behavior is only observed on Chrome and works fine on Firefox.

* I was wondering what could be the reason for this, maybe some issue on Chrome's part, but would like to understand why.

**Comment by mao-sz** [comment]
This isn't something we can do anything about since it's a problem on MDN's side due to a [bug with how Chromium browsers handle certain scroll features](https://github.com/mdn/yari/issues/8049#issuecomment-2325819802). Looks like a fix for Chrome will be coming in a future version, so it shouldn't be an issue for too much longer!

**Reply by kesava-karri** [reply]
Thanks for the reply. and yeah I understand it's something not on our end & thus the reason for discussion rather than raising an issue.
Thank you for the link to the bug, I hope they most likely fix it in the near future.

Going ahead and closing the discussion.
